### PR TITLE
feat: show selected language in button

### PR DESCRIPTION
**Pull request by GeronimoJohn**
To improve the user experience and to let our user know that a language is currently applied or selected. We are to show in the button the current selected languages. If there are multiple languages selected we are to show the top most selected value from the refinements

### CODE REVIEW

File: libs/journeys/ui/src/components/SearchBar/LanguageButton/LanguageButton.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    const button = screen.getByRole('button', { name: 'Language' })
+    fireEvent.click(button)
+    expect(onClick).toHaveBeenCalled()
+  })
+
+  it('should render the button with the selected language', () => {
+    render(<LanguageButton onClick={jest.fn()} selectedLanguage="English" />)
+    expect(screen.getByRole('button', { name: 'English' })).toBeInTheDocument()
+  })
```
Comment (scottkelso) [inline libs/journeys/ui/src/components/SearchBar/LanguageButton/LanguageButton.spec.tsx:24]: Can you add at least one more test test to cover any weird selectedLanguage values? Empty string, multiple commas, no commas but multiple words eta? How should the button render those.

File: libs/journeys/ui/src/components/SearchBar/SearchDropdown/SearchbarDropdown.spec.tsx [hunk 2]
```diff
@@ -111,15 +99,9 @@ describe('SearchbarDropdown', () => {
...
-    const refine = jest.fn()
-    mockUseRefinementList.mockReturnValue({
-      items: languageRefinements,
-      refine
-    } as unknown as RefinementListRenderState)
-
     render(
       <MockedProvider mocks={[getLanguagesContinentsMock]}>
-        <SearchbarDropdown open />
+        <SearchbarDropdown open refinements={refinements} />
```
Comment (scottkelso) [inline libs/journeys/ui/src/components/SearchBar/SearchDropdown/SearchbarDropdown.spec.tsx:104]: Should probably have at least one test that tests usecases for changing language button depending on refinements. I.e. no refinements = `Language`, Spanish = `Spanish`, Spanish and English = `English`

**Review by scottkelso — CHANGES_REQUESTED** [review]
Looking good. Noticed when you tick the Spanish refinement, there is a comma showing on the button. Probably need to sort that. Also maybe one or two additional component tests needed.

**Comment by GeronimoJohn** [comment]
redundant pr
